js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useAssetTagFilter.tsx

```diff
@@ -81,6 +81,6 @@ export function doesFilterArrayMatchValueArray<T, V>(
   return !filterArray.some(
     (filterTag) =>
       // If no asset tags match this filter tag return true
-      !valueArray.find((value) => !isMatch(filterTag, value)),
+      !valueArray.find((value) => isMatch(filterTag, value)),
```

Comment:
this function is used by other filters, can you confirm those still work?

Reply:
I think its only used for owners and tags, this implementation works for both